website/source/docs/builders/ucloud-uhost.html.md

```diff
@@ -0,0 +1,117 @@
+---
+description: |
+    The `ucloud-uhost` Packer builder plugin provides the capability to build
+    customized images based on an existing base images.
+layout: docs
+page_title: 'UCloud Image Builder'
+sidebar_current: 'docs-builders-ucloud-uhost'
+---
+
+# UCloud Image Builder
+
+Type: `ucloud-uhost`
+
+The `ucloud-uhost` Packer builder plugin provides the capability to build
+customized images based on an existing base images.
+
+This builder builds an UCloud image by launching an UHost instance from a source image,
+provisioning that running machine, and then creating an image from that machine.
+
+## Configuration Reference
+
+The following configuration options are available for building UCloud images. They are
+segmented below into two categories: required and optional parameters.
+
+In addition to the options listed here, a
+[communicator](../templates/communicator.html) can be configured for this
+builder.
+
+\~&gt; **Note:** The bulider doesn't support Windows images for now and only supports CentoOS and ubuntu images via SSH anthentication with `ssh_username` (Required) and `ssh_password` (Optional).  The `ssh_username` must be `root` for CentOS images and `ubuntu` for Ubuntu images. The `ssh_password` may contain 8-30 characters, and must be consisted with at least 2 items out of the capital letters, lower case letters, numbers and special characters. The special characters include <code>`()~!@#$%^&*-+=_|{}\[]:;'<>,.?/</code>.
```

Comment:
Okay, so if it _has to_ have all of the things listed, the most idiomatic way to say that in English is 
"and must consist of at least 2"... instead of "and must be consisted with at least 2"